aea/aea.py

```diff
@@ -148,13 +153,25 @@ def __init__(
         )
         self.runtime.set_decision_maker(decision_maker_handler)
 
+        default_ledger_id = (
+            default_ledger
+            if default_ledger is not None
+            else identity.default_address_key
+        )
+        ledger_id_to_currency_denom = (
+            ledger_id_to_currency_denom
+            if ledger_id_to_currency_denom is not None
+            else DEFAULT_LEDGER_ID_TO_CURRENCY_DENOM
```

Comment:
Maybe the default should be an empty dict

Reply:
why?